### ISSUE TITLE
Unable to create new instance groups

### ISSUE THREAD

**Issue by georgebuckerfield** [kubernetes/kops#4303]
Thanks for submitting an issue! Please fill in as much of the template below as
you can.

------------- BUG REPORT TEMPLATE --------------------

1. What `kops` version are you running? The command `kops version`, will display
 this information.
```
$ kops version
Version 1.8.0 (git-f749aa5)
```
2. What Kubernetes version are you running? `kubectl version` will print the
 version if a cluster is running or provide the Kubernetes version specified as
 a `kops` flag.
`1.8.4`
3. What cloud provider are you using?
AWS

4. What commands did you run?  What is the simplest way to reproduce this issue?
Create a basic cluster and attempt to add a new instance group:
```
kops create ig nodes-extra
```

5. What happened after the commands executed?
```
I0119 17:02:45.473342   18419 s3fs.go:198] Reading file "s3://<bucket_location>/instancegroup/nodes-extra"

InstanceGroup.kops "nodes-extra" not found
```

6. What did you expect to happen?
To drop into an editor for the new instance group.
7. Please provide your cluster manifest. Execute
  `kops get --name my.example.com -oyaml` to display your cluster manifest.
  You may want to remove your cluster name and other sensitive information.

8. Please run the commands with most verbose logging by adding the `-v 10` flag.
  Paste the logs into this report, or in a gist and provide the gist link here.

9. Anything else do we need to know?

The error is coming from this line in `create_ig.go`:
```
existing, err := clientset.InstanceGroupsFor(cluster).Get(groupName, metav1.GetOptions{})
```
I think this is due to a change introduced in https://github.com/kubernetes/kops/pull/3981. Now that errors are actually being returned by this line: https://github.com/kubernetes/kops/blob/f749aa5b0c742aad0ef8ac6bab243365ddd4219f/pkg/client/simple/vfsclientset/instancegroup.go#L104
I think it's blocking the rest of the `create ig` command from executing.

**Comment by georgebuckerfield** [comment]
I have a fix for this locally, I'll try and get a PR submitted tonight.

**Comment by chrislovecnm** [comment]
@georgebuckerfield there have been multiple reports for this issue, can you round up the issues so we close them all?

**Comment by gambol99** [comment]
A similar issue was fixed in https://github.com/kubernetes/kops/pull/4275 .. as errors.NewNotFound was introduced

**Comment by georgebuckerfield** [comment]
Looking at other issues around instance groups, I think #4209 can be closed as well, that was fixed in #3987.
